runners/spark/src/main/java/org/apache/beam/runners/spark/SparkJobServerDriver.java

```diff
@@ -36,6 +36,7 @@
 
   /** Spark runner-specific Configuration for the jobServer. */
   public static class SparkServerConfiguration extends ServerConfiguration {
+
```

Comment:
Nit: remove whitespace changes to this file.

Reply:
Done.